### DISCUSSION TITLE
Visidata has over 6K GitHub Stars!

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#1548] [General]
🎆 Congratulations, @saulpw!  And  @anjakefala!  

Also, there was a big jump of about 300 Twitter followers recently.  I'm not sure what caused that.

**Comment by saulpw** [comment]
Wow!! I only check these metrics (particularly the Twitter followers occasionally), so thanks for making a note of it.  We've had some high exposure on Twitter recently thanks to @simonw and @metalx1000 so that's probably where those came from.

BTW @frosencrantz, would you like any VisiData stickers?  You've been quite helpful in answering peoples' questions and fostering community, and I'd love to honor that contribution.

**Reply by metalx1000** [reply]
Thank you for Visidata.  I feel like I have just scratched the surface of what it can do.  It's such an awesome program.  Keep up the good work.

**Comment by frosencrantz** [comment]
@saulpw. Thank you for the offer of the stickers! 😄 I'm good for now.  I do hope we can have a helpful community around VisiData.  I think that will be very beneficial to everyone.

Also, thank you @metalx1000 for putting your video up on YouTube!

P.S. @saulpw  if you ever get something like xkcd.com/208 that stars VisiData lets talk...  😄 🦸

**Comment by frosencrantz** [comment]
@saulpw  I agree these metric milestones are arbitrary.  I'm not sure how they translate to something that is more interesting like supporters, or contributors.

The milestones do provide an excuse to celebrate and might provide a way to help on other goals.  Will McGugan has been using his Twitter follower milestones as an excuse to provide exposure to other projects.  

I've been wondering if it is possible to have a wider base of VisiData plugins, there are already a small set of great ones from you and others .  Like Vim has many plugins available, even many that do similar things, VisiData is far away from that.  Or a few years ago you started a thread on Twitter about rough edges, getting community feedback.

 It really depends on what you might want from this project and community.

**Comment by frosencrantz** [comment]
I guess I'll add that a new anniversary is around the corner of 6 years.
